frontend/templates/troubleshooting/Resource.tsx

```diff
@@ -184,20 +216,44 @@ function Resource({
    const { themeColor, iconColor, icon } = difficultyTheme;
    const breakpoint = useBreakpoint();
    const isMobile = breakpoint === 'base';
+   const buyButtonText =
+      typeof showBuyButton === 'string' ? showBuyButton : 'Buy';
 
    return (
       <ResourceBox>
          {imageUrl && (
-            <Image
+            <Box
+               position="relative"
+               mr={2}
                boxSize="64px"
+               minWidth="64px"
+               borderRadius="md"
                outline="1px solid"
                outlineColor="gray.300"
-               borderRadius="md"
-               objectFit="cover"
-               alt={title}
-               src={imageUrl}
-               mr={2}
-            />
+            >
+               {showStackedImages && (
+                  <Box
+                     width="100%"
+                     height="100%"
+                     borderRadius="inherit"
+                     outline="inherit"
+                     outlineColor="inherit"
+                     position="absolute"
+                     top="0"
+                     transform="rotate(6deg)"
+                  />
```

Comment:
So this is meant to be empty? Might want to add a comment clarifying that we're faking a stack of images.

Reply:
[db7d85b](https://github.com/iFixit/react-commerce/pull/2104/commits/db7d85b6220883af1d88545d1abf9818499b6930)